app/src/main/java/com/besome/sketch/design/DesignActivity.java

```diff
@@ -227,6 +233,8 @@ private void indicateCompileErrorOccurred(String error) {
                 startActivity(intent);
             }
         });
+        /* Set the text color to yellow */
+        snackbar.setActionTextColor(Color.YELLOW);
```

Comment:
why did you added this?

Reply:
I don't know 🥲, I think I am not added it

Reply:
Remove it